src/SDDPoptimize.jl

```diff
@@ -135,25 +135,29 @@ function run_SDDP!(model::SPModel,
         callsolver_backward = backward_pass!(model, param, V, problems, stockTrajectories, model.noises)
 
         ####################
+        # Update stats
```

Comment:
This solution works, otherwise we should use an incremental counter outside stats object.